spec/requests/inventories_spec.rb

```diff
@@ -0,0 +1,93 @@
+require 'rails_helper'
```

Comment:
- [ ] Greate job using Capybara testing libraries for writing integration test now let's add unit test as well using RSpec 👍🏻